### ISSUE TITLE
Reduce the minimum fee?

### ISSUE THREAD

**Issue by billym2k** [dogecoin/dogecoin#1758]
If Dogecoin is meant to be used more frequently for small sends, I think the fee is a bit high at the moment - though arguably this is due to the current fiat value - would probably want to wait awhile to make this change as the fiat value is super volatile. If the fee is too low, we risk people spamming the blockchain with inexpensive + pointless dust transactions, but if the fee is too high it reduces the utility of the coin.

What do y'all think of the current fee of 1 DOGE?

(I think to change it, the code is in validation.h, can change `static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = COIN;` to `static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = CENT;` to reduce it 100x for example)

**Comment by DemonRx** [comment]
This brings up an interesting point. Has the network ever been stress tested ?

**Comment by cipig** [comment]
https://github.com/KomodoPlatform/atomicDEX-API/issues/829

**Comment by patricklodder** [comment]
Hi! There is a discussion about this on reddit dogecoindev sub, started by @shibe2 a little under 2 weeks ago: 

https://www.reddit.com/r/dogecoindev/comments/lg4hp7/reducing_dogecoin_transaction_fees/

TLDR from my perspective is: 

1. Remember that fees are linked to DUST limit, and the latter is probably more important than the former... so we definitely need to look at both. 
2. As Billy said, the price is still highly volatile, so any calculation of how many fiat coins one is spending on Dogecoin txs, can be invalid tomorrow. Since recommendations through our software release process are definitely slow, making decisions now that can be changed tomorrow are risky.
3. I do think that we should - as soon as the volatility lessens at least a bit - re-assess the recommended fees to make more sense for those shibes that do not share the `1D = 1D` vision, and make sure that they make sense for everyone.

**Comment by billym2k** [comment]
@patricklodder thanks for that link! I'm glad to see the discussion on this - would the preference for discussions like this be on the dogecoindev subreddit or through github issues?

**Comment by patricklodder** [comment]
I'd personally prefer the (emotional) debates to be on reddit, so that everyone's inbox stays clean of a million one-liner replies, and then we can then have a final go at it, when there is a proposal / direction and a thought out plan here.

**Comment by billym2k** [comment]
Sounds good! I'll close this issue out then - I think the 'let's see where this all shakes out in x-amount of time' is the most sane approach!
